source/Octopus.Client/Editors/Async/TenantEditor.cs

```diff
@@ -39,6 +39,28 @@ public async Task<TenantEditor> CreateOrModify(string name)
 
             return this;
         }
+        
+        public async Task<TenantEditor> CreateOrModify(string name, string description, string cloneId = null)
```

Comment:
I think we should add `param` comments.  As a user I possibly don't know what `cloneId` is.

Reply:
Even though xml comments are evil... I concur 👍